### ISSUE TITLE
{   "code": 400,   "message": "Payload contains invalid json" }

### ISSUE THREAD

**Issue by KoRba88** [copercini/esp32-iot-examples#14]
Hi

When my ESP32 want to Publish message: Hello world, In the activity tab on my thing I only see this messege

{
  "code": 400,
  "message": "Payload contains invalid json"
}

**Comment by copercini** [comment]
probably your message has a problem with json formatting

**Comment by KoRba88** [comment]
Yes, I know it but why? I didn't made any changes in the provided example sketch so it should work.

Also using MQTT.fx client i see this messege and i can publish nothing.

**Comment by copercini** [comment]
Sorry for the late answer, it's sending `Hello world 1` as message that isn't JSON formatted

you can fix it changing [this line](https://github.com/copercini/esp32-iot-examples/blob/master/ESP32_aws_iot/ESP32_aws_iot.ino#L163) to `snprintf (msg, 75, "{\"Hello world\":\"#%ld\"}", value);`
